lib/models/comment/comment_model.dart

```diff
@@ -0,0 +1,91 @@
+class CommentModel {
+  CommentModel({
+    this.type,
+    this.comment,
+    this.user,
+    this.createdAt,
+    this.updatedAt,
+    this.id,
+  });
+
+  String? type;
+  String? comment;
+  User? user;
+  DateTime? createdAt;
+  DateTime? updatedAt;
+  String? id;
+
+  factory CommentModel.fromJson(Map<String, dynamic> json) => CommentModel(
+        type: json["type"],
+        comment: json["comment"],
+        user: User.fromJson(json["user"]),
+        createdAt: DateTime.parse(json["created_at"]),
+        updatedAt: DateTime.parse(json["updated_at"]),
+        id: json["id"],
+      );
+
+  Map<String, dynamic> toJson() => {
+        "type": type,
+        "comment": comment,
+        "user": user?.toJson(),
+        "created_at": createdAt?.toIso8601String(),
+        "updated_at": updatedAt?.toIso8601String(),
+        "id": id,
+      };
+}
+
+class User {
+  User({
+    this.username,
+    this.profileImg,
+    this.id,
+  });
+
+  String? username;
+  ProfileImg? profileImg;
+  String? id;
+
+  factory User.fromJson(Map<String, dynamic> json) {
+    return User(
+      username: json["username"],
+      profileImg: ProfileImg.fromJson(json["profile_img"]),
```

Comment:
Did this auto generate? but I think it will get errors when profile_img is null, we should check if it is null before putting it in ProfileImg.fromJson(..)